docs/source/concept_models/vivarium_csu_multiple_myeloma_2/concept_model.rst

```diff
@@ -147,15 +141,35 @@ set of treatment categories based on treatment guidelines from the `NCCN
 5.0 Simulation Scenarios
 ++++++++++++++++++++++++
 
+We know that our scenarios will roughly correspond to:
+
+* Scenario 0: Naive treatment assignment
+* Scenario 1: baseline/low Isa uptake (e.g., 0% uptake of Isa following Dara frontline treatment)
+* Scenario 2: 5% uptake of Isa following Dara frontline treatment
+* Scenario 3: 5% uptake of Isa frontline treatment (e.g, instead of Dara frontline treatment, replacing a certain 'other' treatment regimen, etc.)
+
+.. todo::
+
+  Rigorously define these scenarios.
+
 6.0 Simulation Parameters
 +++++++++++++++++++++++++
 
 6.1 Locations
 -------------
 
+United States.
+
 6.2 Population and Randomness
 -----------------------------
 
+Population description:
+
+* Cohort type: Prospective closed cohort of individuals aged 15 years and older (open cohort if we only examine on age 50+ population)
+* Size of largest starting population: 100,000 simulants
+* Time span: Jan 1, 2013 to Dec 31, 2027 (Jan 1, 2013 to Jan 1, 2023 is a 10-year long burn-in period)
+* Time step: 90 days
```

Comment:
Note: Phase 1 was 28 days, I don't _think_ anything heavily relied on the time step duration, and we have talked about changing to 90 to help with runtime.